qendpoint-backend/pom.xml

```diff
@@ -46,12 +46,23 @@
         <rdf4j.version>5.0.2</rdf4j.version>
         <spring.version>3.4.0</spring.version>
         <logback.version>1.5.6</logback.version>
+        <jackson.version>2.18.1</jackson.version>
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
     <dependencies>
+        <dependency>

```

Comment:
Runtime issue actually. Complained of a method not found when generating the query explanation json. The project inherits two different versions of Jackson, so we apparently need to decide which one to use.

It would be cleaner to use dependency management instead, but I couldn't find that in any of the poms. Maybe I overlooked something?